testsuite/grpc-tests/src/test/java/org/jboss/resteasy/test/grpc/GrpcListsAndSetsTest.java

```diff
@@ -0,0 +1,973 @@
+package org.jboss.resteasy.test.grpc;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.GenericType;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.protobuf.Any;
+import com.google.protobuf.Message;
+
+import dev.resteasy.grpc.arrays.Array_proto;
+import dev.resteasy.grpc.bridge.runtime.Utility;
+import dev.resteasy.grpc.example.CC1;
+import dev.resteasy.grpc.example.CC1JavabufTranslator;
+import dev.resteasy.grpc.example.CC1ServiceGrpc;
+import dev.resteasy.grpc.example.CC1_Server;
+import dev.resteasy.grpc.example.CC1_proto;
+import dev.resteasy.grpc.example.CC1_proto.*;
+import dev.resteasy.grpc.example.CC1_proto.GeneralEntityMessage;
+import dev.resteasy.grpc.example.CC1_proto.GeneralReturnMessage;
+import dev.resteasy.grpc.example.sub.CC8;
+import dev.resteasy.grpc.lists.sets.D1;
+import dev.resteasy.grpc.lists.sets.D3;
+import dev.resteasy.grpc.lists.sets.DD1;
+import dev.resteasy.grpc.lists.sets.L3;
+import dev.resteasy.grpc.lists.sets.S1;
+import dev.resteasy.grpc.lists.sets.S3;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+@SuppressWarnings("deprecation")
+@RunWith(Arquillian.class)
+@RunAsClient
+public class GrpcListsAndSetsTest {
+
+    private static CC1JavabufTranslator translator = new CC1JavabufTranslator();
+    private static ManagedChannel channelPlaintext;
+    private static CC1ServiceGrpc.CC1ServiceBlockingStub blockingStubPlaintext;
+    private static Set<String> entityClasses = new HashSet<String>();
+    private static Map<String, Class<?>> CLASS_MAP = new HashMap<String, Class<?>>();
```

Comment:
This is never used.

Reply:
Actually, it looks like the first three are used. CLASS_MAP is gone. Ah, maybe that's what you meant.